tests/Unit/Message/JsonMessageSerializerTest.php

```diff
@@ -113,7 +115,7 @@ public function testSerialize(): void
         $json = $serializer->serialize($message);
 
         $this->assertEquals(
-            '{"name":"handler","data":"test","meta":[]}',
+            '{"name":"handler","data":"test","meta":[],"class":"Yiisoft\\\\Queue\\\\Message\\\\Message"}',
```

Comment:
Can we use a map of `name => class` somewhere (probably hardcoded in the envelope itself)? I'd avoid PHP-specific FQNs at all costs.

Reply:
This class is specific to an application which pushes a message to a queue. This class list is not usable by any other service. I'll move this class to the "meta" key.